### PR TITLE
Add supported firmware version to README.

### LINKED ISSUE #369: Add supported firmware versions to README
Version 1.1.0 should be the only supported version.

### PR DESCRIPTION
**Pull request by SansoneG**
- [ ] Add command to check for current firmware version?
- [ ] Clarify correct naming of `PRBT firmware version 1.1.0`

### CODE REVIEW

**Comment by martiniil** [comment]
>     * [ ]  Add command to check for current firmware version?

Not required here.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ sudo apt install ros-melodic-pilz-robots
 `melodic-devel` is considered to be the active development branch.
 Relevant changes are cherry-picked into `kinetic-devel` on a case-by-case basis.
 
+### Supported hardware versions
+PRBT firmware version 1.1.0
+
+If you want to update your PRBT firmware please contact our customer support at ros@pilz.de.
```
Comment (martiniil) [inline README.md]: I don't know if this is the correct address for this purpose.

**Review by martiniil — CHANGES_REQUESTED** [review]
Needs clarification. See comment.

File: README.md (hunk 1 above)
Comment (SansoneG) [reply README.md]: Clarified with @jschleicher

**Review by martiniil — APPROVED** [review]

**Pull request merged**